src/gigs-board/components/molecule/markdown-viewer.jsx

```diff
@@ -0,0 +1,54 @@
+const Wrapper = styled.div`
+  p {
+    white-space: pre-line; // This ensures text breaks to new line
+
+    span {
+      white-space: normal; // and this ensures profile links look normal
+    }
+  }
+
+  blockquote {
+    margin: 1em 0;
+    padding-left: 1.5em;
+    border-left: 4px solid #ccc;
+    color: #666;
+    font-style: italic;
+  }
```

Comment:
Can we also make the font size the same as a regular text?

Currently, blockquote text is bigger than the regular text:

<img width="829" alt="image" src="https://github.com/near/neardevhub-widgets/assets/304265/f06eae3d-66e2-474e-a61d-ec18bc1637e9">

Reply:
Here you go!

<img width="239" alt="Screenshot 2023-09-10 at 10 23 51 AM" src="https://github.com/near/neardevhub-widgets/assets/16282460/c763ac20-909f-40e5-9900-6a862bfc6d88">